src/index.tsx

```diff
@@ -205,8 +237,20 @@ let recursiveUrlCollection = (bookmark: chrome.bookmarks.BookmarkTreeNode, urlLi
     }
 }
 
-var showNotice = (message: string): void => {
-    //TODO: implement showNotice
+var showNotice = (notificationId: constants.NotificationId, title: string, message: string = ''): void => {
+    chrome.notifications.create(
+        notificationId, // prevents duplicate notifcations (only keeps the most recent one)
+        {
+            'type': 'basic', // required
+            'iconUrl': 'images/default.png', // required
+            'title': title, // required
+            'message': message, // required
+            'eventTime': Date.now(),
+        },
+        function (notificationId: string) {
```

Comment:
what would we do in this callback if we remove all the console logs later on?

Reply:
Oh, good point. I this callback isn't necessary, but because this is my first time using this API, I wanted to try all the optional ones. We can safely delete it then.

Reply:
good, let's make an issue or note so that we don't forget this